src/test/java/teammates/test/cases/logic/StudentsLogicTest.java

```diff
@@ -431,56 +431,61 @@ public void testEnrollLinesChecking() throws Exception {
                     + Const.EOL + lineWithInvalidEmail + Const.EOL + lineWithInvalidStudentNameAndEmail + Const.EOL
                     + lineWithInvalidTeamNameAndEmail + Const.EOL + lineWithInvalidTeamNameAndStudentNameAndEmail;
 
-        invalidInfo = getInvalidityInfoInEnrollLines(enrollLines, courseId);
+        invalidInfoString = null;
```

Comment:
Oh actually it is not needed to be reset as signalFailureToDetectException() already checks for exception and invalidInfoString must not be null.(I used assert not null to test exception and forgot to delete this line after my update). However, the null is assigned to initial value to suppress warning by compiler and null is assigned when test for
         ______TS("enrollLines with correct input");
As exception is not caught and I have to use assertNull to check for that.